app/Project.php

```diff
@@ -107,6 +137,26 @@ public function setConfig($config)
             $this->enable_contact = (bool)$config['enable_contact'];
         }
 
+        if (isset($config['enable_connection'])) {
```

Comment:
I do not thing we have to add so much configuration parameters. Why would we activate user profile without connection? Or without admin?
I guess a single configuration parameter to enable profiles creation should activate profile, connection, admin, ... Another parameter for registration may have sense.